### DISCUSSION TITLE
Request app-ads.txt static file inside public folder always return 404?

### DISCUSSION THREAD

**Discussion by RafilAlArsApps** [vercel/vercel#5679] [Help]
Create an app-ads.txt file that includes the authorized sellers for my google ads, then publish it at mydomain.com/app-ads.txt always return 404, when I changed the file name to something else working but with this name never work I do not know why that happening anybody could help please.

I put the file inside public folder.

**Comment by paulogdm** [comment]
I could not reproduce the issue:

- File on https://memory-test-bzdslce86.vercel.app/app-ads.txt
- Source available at https://memory-test-bzdslce86.vercel.app/_src

**Reply by RafilAlArsApps** [reply]
Finally the issue has been resolved, it was due to mac computer change the type of txt file to something else which is not standard. 

Thanks everybody for help.

**Comment by naterexw** [comment]
@RafilAlArsApps 
Double check your file is not in `/public/static` folder.  It needs to be in `/public` folder!

**Reply by RafilAlArsApps** [reply]
yes it is in the public folder, and the strange thing when I rename the file everything work perfect.
